vllm/model_executor/layers/quantization/fp8.py

```diff
@@ -203,7 +207,7 @@ def __init__(self, quant_config: Fp8Config):
         self.use_marlin = (not current_platform.has_device_capability(89)
                            or envs.VLLM_TEST_FORCE_FP8_MARLIN)
         # Disable marlin for rocm
-        if current_platform.is_rocm():
+        if current_platform.is_rocm() or current_platform.is_xpu():
```

Comment:
Should we say `not is_cuda` here?

```suggestion
        if not current_platform.is_cuda()
```